app/shared/ui-subject/src/commonMain/kotlin/ui/subject/details/state/SubjectDetailsStateFactory.kt

```diff
@@ -92,6 +109,21 @@ class DefaultSubjectDetailsStateFactory : SubjectDetailsStateFactory, KoinCompon
 
     val sessionManager: SessionManager by inject()
 
+    /**
+     * 已经把条目详情信息缓存到本地数据库的条目 ID.
+     *
+     * 对于已经缓存过的条目, 创建 SubjectDetailsState flow 时不会 emit [预加载][SubjectDetailsState.showPlaceholder] 条目.
+     */
+    private val cachedSubjectDetails: MutableStateFlow<List<Int>> = MutableStateFlow(emptyList())
```

Comment:
```suggestion
    private val cachedSubjectDetails: MutableStateFlow<IntList> = MutableStateFlow(intListof())
```
听起来会浪费内存 (毕竟会有数千个对象, 而且会永久放在内存里, 也就会导致 0.1MB 的内存占用). 反正换一下 IntList 就能节约了, 我觉得可以节约一下